src/__test__/components/SeedlotActivities.test.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import SeedlotActivities from '../../components/SeedlotActivities';
+import '@testing-library/jest-dom';
+
+describe('Seedlot Activities component', () => {
+  beforeEach(() => {
+    render(<BrowserRouter><SeedlotActivities /></BrowserRouter>);
```

Comment:
Break this render in multiple lines, to maintain the same pattern as other tests suits and for better readability :)